packages/release/proxy-env

```diff
@@ -0,0 +1,6 @@
+{{#if settings.network.https-proxy}}
+HTTPS_PROXY={{settings.network.https-proxy}}
+https_proxy={{settings.network.https-proxy}}
+{{/if}}
+NO_PROXY={{#each settings.network.no-proxy}}{{this}},{{/each}}localhost,127.0.0.1{{#if settings.kubernetes.api-server}},{{host settings.kubernetes.api-server}},.svc,.default,.local,.{{settings.kubernetes.cluster-domain}}{{/if}}
```

Comment:
Are we making queries to a `.svc`, `.default`, or `.local` TLD? 

I'm wondering why we don't have this instead:
```suggestion
NO_PROXY={{#each settings.network.no-proxy}}{{this}},{{/each}}localhost,127.0.0.1{{#if settings.kubernetes.api-server}},{{host settings.kubernetes.api-server}}{{/if}}{{#if settings.kubernetes.cluster-domain}},.{{settings.kubernetes.cluster-domain}}{{/if}}
```

Reply:
You're right. I probably don't need those. I originally thought I had to `no_proxy` kubernetes service domains with `.svc`, `.default` but they're all suffixed with the cluster domain anyways.

I'll go ahead and remove those.